client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/usecase/LoginUserUseCase.kt

```diff
@@ -0,0 +1,46 @@
+package org.thechance.common.domain.usecase
+
+import org.thechance.common.domain.entity.UserTokens
+import org.thechance.common.domain.getway.ILocalGateway
+import org.thechance.common.domain.getway.IRemoteGateway
+
+interface ILoginUserUseCase {
```

Comment:
I think we need to add the method for store the local flag ( keep me logged in) also